examples/examples_transitive_deps.bzl

```diff
@@ -8,6 +8,11 @@ load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
 load("@build_bazel_rules_nodejs//:index.bzl", "node_repositories")
 load("@rules_proto//proto:repositories.bzl", "rules_proto_dependencies", "rules_proto_toolchains")
 
+load("@crate_universe_basic_rust_deps//:defs.bzl", basic_pinned_rust_install = "pinned_rust_install")
```

Comment:
Can the `cargo_universe` be it's own workspace that's `.bazelignore`'d by the top level examples workspace? I think it'd make the examples clearer for users, though don't feel too strongly about this one, up to you.

Reply:
Yeah, I struggled with this - I think it's not super clear now, but also, a lot of effort has gone into making both `@rules_rust//examples/...` and `@examples/...` work, and it felt like adding a third repo makes this harder still... That said, I'm happy to re-work however folks think is clearest

Reply:
The thing I care most about is having the repository definitions colocated with the examples. It's not immediately clear that the definitions are in `examples_deps.bzl`.

Bazel's ability (or inability) to handle transitive workspaces definitely makes me sad, but I think it's important to isolate examples as much as possible. I find they become less valuable the more they become a dropbox of random code. Not saying that's the case now, just my fear that things escalate to that point.